protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3456,3 +3477,134 @@ message LiquidationPrice {
   // Liquidation price assuming sell orders start getting filled.
   string including_sell_orders = 3;
 }
+
+// Team record containing the team information.
+message Team {
+  // Unique identifier of the created team.
+  string team_id = 1;
+  // Party ID that created the team.
+  string referrer = 2;
+  // Optional team name to be added to the referral banner.
+  optional string name = 3;
+  // Optional link to a team forum, Discord, etc.
+  optional string team_url = 4;
+  // Optional link to an image to be used as the team avatar.
+  optional string avatar_url = 5;
+  // Time in Unix nanoseconds when the team is created.
+  int64 created_at = 6;
+}
+
+// Team data item with the corresponding cursor.
+message TeamEdge {
+  // Team data
+  Team node = 1;
+  // Cursor that can be used to fetch further pages.
+  string cursor = 2;
+}
+
+// Page of team data and corresponding page information.
+message TeamConnection {
+  // Page of team data and their corresponding cursors.
+  repeated TeamEdge edges = 1;
+  // Page information that is used for fetching further pages.
+  PageInfo page_info = 2;
+}
+
+// Request to list all teams
+message ListTeamsRequest {
+  // Optional team id to filter for a specific team
+  optional string team_id = 1;
```

Comment:
Funnily enough I had it that way originally, but I thought it would be easier/neater for users to have just one, anyone from FE have an opinion on this?